lib/game_teams_collection.rb

```diff
@@ -13,21 +13,66 @@ def initialize(file_path)
   def create_game_teams_array(file_path)
     load_from_csv(file_path, GameTeams)
   end
-  # require "pry"; binding.pry
-end
-
 
+  def game_teams_hash
+    @game_teams_array.reduce({}) do |hash, game_teams|
+      hash[game_teams.team_id] << game_teams if hash[game_teams.team_id]
+      hash[game_teams.team_id] = [game_teams] if hash[game_teams.team_id].nil?
+      hash
+    end
+  end
 
-  def game_teams_by_id
+  def percentage(numerator, denominator) #to-do: make Calculatable module
```

Comment:
I like this. When we refactor more could also build average methods.

Reply:
yes